### DISCUSSION TITLE
Why X in asphyxiate is not split into k and s?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#23] [Q&A] [answered]
In the dictionary - asphyxiate: A/STP*EUBGS/KWRAEUT, AS/TP*EUBGS/KWRAEUT

x is entered by keys -BGS, but in the study material x is split into k and s across two syllables unless x is followed by a consonant.

Is this an exception, or an oversight or I've interpreted the lesson incorrectly?

**Comment by aerickt** [comment]
`AS/TP*EUBG/SEU/KWRAEUT` and `A/STP*EUBG/SEU/KWRAEUT` would be correct too.  However, 4 strokes is quite long. Chapter 15 goes over a new technique where the -iate sound can be represented in one stroke as `KWRAEUT`. Overall, the `EU` sound in the third stroke of  `A/STP*EUBG/SEU/KWRAEUT` can be represented by the `KWR` chord in the last stroke, and the `S` sound can be moved to the previous stroke by keeping the x sound in one chord.

`A/STP*EUBGS/KWRAEUT`
a/sphyx/iate (you can approximate this like "a-sphix-yate")

**Comment by evgenyfadeev** [comment]
Chat GPT says that -iate is a suffx - in that case, after removing the suffix x is the last character in the syllable and therefore it should not be slpit.

AS/TP*EUBG/SEU/KWRAEUT and A/STP*EUBG/SEU/KWRAEUT are not in the dictionary, btw.

**Reply by aerickt** [reply]
I suggest looking over https://lapwing.aerick.ca/Chapter-15.html#as-a-vowel-pair .

Yes, it is true that we should normally avoid splitting words in a way that forces us to use `KWR` in the next stroke. However, if we followed this everywhere, we would have to write "asphyxiate" in 4 strokes (too many) Therefore, it is acceptable to represent -ia- glides as `KWRA` or `KWRAEU` to cut down the number of strokes (since a sound normally represented with two strokes and two vowels can now be compressed into one stroke).

`AS/TP*EUBG/SEU/KWRAEUT` and A/STP*EUBG/SEU/KWRAEUT are indeed correct, but are discouraged because they are too long of an outline. Furthermore, they are not always present in the dictionary as you have found out.

**Reply by aerickt** [reply, marked as answer]
Also, I would not consider -iate a suffix in this case since "asphyx" is not really a word. We're not using `KWRAEUT` as a suffix, but rather it is a better way to split the word syllabically since it allows us to save on strokes.

**Reply by evgenyfadeev** [reply]
I was mostly confused about not having to split the -x- into -k- and -s- since there is no following consonant.

but I guess KWR is a silent consonant, so then not splitting X makes sense, even if -ciate is not a suffix in this case.

**Reply by aerickt** [reply]
I would think about `KWR` as a 'y' sound in this word rather than silent.

**Reply by evgenyfadeev** [reply]
but then according to the rules X needs to be split into k and s. I'm trying to find the justification that would not violate the rules in CH15. If I consider KWR a silent consonant that also adds a gliding y before the a, then all makes sense, otherwise I don't see an explanation for not breaking up the X.

**Reply by evgenyfadeev** [reply]
btw - I find the syllabic splitting the hardest part so far. on the vowels I was getting about 50% correctness on the first pass and on this about 30%.

**Reply by aerickt** [reply]
> but then according to the rules X needs to be split into k and s. I'm trying to find the justification that would not violate the rules in CH15. If I consider KWR a silent consonant that also adds a gliding y before the a, then all makes sense, otherwise I don't see an explanation for not breaking up the X.

X does not *always* need to be split into k and s. You can split a word at an x if there is no other way, but you don't have to. Perhaps I can add a note in https://lapwing.aerick.ca/Chapter-15.html?highlight=x#exam to clarify that?

**Reply by evgenyfadeev** [reply]
A note would be useful indeed. But how would you know when to split and when not to? It would be nice to have a rule.

**Reply by aerickt** [reply]
Usually if splitting the x results in more strokes, it's best to keep the it as one. I'm not sure if this applies in situations other than "xia" words like "asphyxiate". But you'd want to split all "xia" like `...-BGS/KWR...`. Splitting the x is basically a last resort if there's no other method.

Although, now that you've brought it up, I suppose I could do away with splitting x's like this and just always keeping it as `KP` or `BGS`.

**Reply by evgenyfadeev** [reply]
My opinion hardly has any weight here, I'm about 1 month into learning steno. It does make sense though - split X into K and S only if you don't add strokes and to avoid adding syllables that start with a vowel.
